### ISSUE TITLE
Remove django from install_requires

### ISSUE THREAD

**Issue by manelclos** [jazzband/django-debug-toolbar#1212]
Hi,

If django-debug-toolbar install is processed before django install, an incorrect version can be downloaded/installed. This is specially annoying when using pip download only.

Would this be considered?

Thanks!

**Comment by matthiask** [comment]
Is there a different way to specify the minimum required Django version (except for runtime checks)?

I often do not add Django to `install_requires` to my own Django packages, but I think something like `python_requires` for dependencies other than Python would be awesome.

Long story short: I'm not sure where I stand on this.

**Comment by manelclos** [comment]
@matthiask I haven't found any way to avoid installing in an old version other that `install_requires`. I found though a way to workaround this issue I opened, and it is to add `--no-deps` when issuing `pip download`.

**Comment by tim-schilling** [comment]
@manelclos Can you provide a way for me to reproduce this?

**Comment by manelclos** [comment]
Hi @tim-schilling!

I initially solved this using --no-deps, but then we found the root problem:
- we have a Dockerfile that would install requirements.txt and optional requirements-devel.txt.
- we were downloading them **separately**, this is, at the time we were processing requirements-devel.txt, pip has no information about Django being downloaded already because of requirements.txt. This produced the download of Django 2.2.6 from requirements.txt and Django 2.2.7 from requirements-devel.txt (due to django-debug-toolbar).
- trying to install all requirements from the download directory failed because of two different Django versions were present.
- we fixed the Dockerfile and in case the requirements-devel.txt also needs to be installed, we download all files at the same time, like `pip download -r requirements.txt -r requirements-devel.txt`

So probably this issue can be closed. I understand the install_requires is now useful to require Django >= 2.0 so developers know django-debug-toolbar version is not compatible with older Django versions.
